bigbluebutton-html5/imports/ui/services/auth/index.js

```diff
@@ -217,6 +218,9 @@ class Auth {
         this.loggedIn = true;
         this.uniqueClientSession = `${this.sessionToken}-${Math.random().toString(36).substring(6)}`;
       })
+      .catch((err) => {
+        logger.warn("Failed to validate token, error: {}", err);
```

Comment:
```suggestion
        logger.error(`Failed to validate token: ${err.description}`);
```

The placeholder `{}` is used in Scala. Not in Javascript.